CRM/Contact/Page/AJAX.php

```diff
@@ -1073,6 +1074,9 @@ public static function getContactRelationships() {
 
     $params['contact_id'] = $contactID;
     $params['context'] = $context;
+        if($relationship_type_id){
```

Comment:
code is not indented correctly. Please refer: http://wiki.civicrm.org/confluence/display/CRMDOC/PHP+Code+and+Inline+Documentation